.github/workflows/alpha_release.yml

```diff
@@ -0,0 +1,49 @@
+name: Create Alpha Release
+on:
+  push:
+    branches:
+      - master
+  
+jobs:
+  build:
+    name: Create Alpha Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Package Version
+      id: package-version
+      uses: codota/npm-get-version-action@master
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Install
+      run: npm ci --no-optional
+    - name: Package
+      run: npm run vsce:package
+    - name: Tag commit
+      id: create_tag
+      uses: codota/github-commit-timestamp-tagger@master
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        base_version: v${{ steps.package-version.outputs.current-version }}-alpha
```

Comment:
This will enable multiple alpha versions?

Reply:
yes, it will create a tag with a timestamp,  prefixed with the `base_version ` ie: v3.1.11-alpha.234234234